tests/nlu/test_train.py

```diff
@@ -129,7 +129,6 @@ def test_all_components_are_in_at_least_one_test_pipeline():
 
     for cls in registry.component_classes:
         if "convert" in cls.name.lower():
-            # TODO
```

Comment:
this is still a TODO, isn't it?

Reply:
thanks for catching that! Accidentally removed it. Its back now.